src/cloud-cli/userdb.ts

```diff
@@ -177,10 +177,29 @@ export function migrate(): number {
     migratecommands?.forEach((cmd) => {
       const command = "npx " + dbType + " " + cmd;
       logger.info("Executing " + command);
-      execSync(command);
+      const migrateCommandOutput = execSync(command).toString();
+      logger.info(migrateCommandOutput);
     });
   } catch (e) {
     logger.error("Error running migration. Check database and if necessary, run npx dbos-cloud userdb rollback.");
+    if (e instanceof Error) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+      const stderr = (e as any).stderr;
+      if (stderr && Buffer.isBuffer(stderr) && stderr.length > 0) {
+        logger.error(`Standard Error: ${stderr.toString().trim()}`);
+      }
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+      const stdout = (e as any).stdout;
+      if (stdout && Buffer.isBuffer(stdout) && stdout.length > 0) {
+        logger.error(`Standard Output: ${stdout.toString().trim()}`);
+      }
+      if (e.message) {
+        logger.error(e.message);
+      }
+    } else {
+      // If 'e' is not an Error object, log it as a generic error
+      logger.error('An unknown error occurred:', e);
+    }
```

Comment:
When we have some time we should have a wrapper around `execSync` which also handles errors.